### DISCUSSION TITLE
Occasional SSL Error Despite SSL Certificate Installed Need Guidance

### DISCUSSION THREAD

**Discussion by LeeKoehn** [community/community#147289] [Programming Help]
### Select Topic Area

Question

### Body

Hi everyone,

I manage a custom packaging website, [London Packaging Solution](https://custompackaginglondon.co.uk/) and although I’ve installed an SSL certificate on the site, I occasionally encounter an error where browsers display a "This website is not secure" warning.

I’ve double checked that the SSL certificate is valid but the issue still appears intermittently. I suspect it could be related to mixed content or server configuration but I’m not entirely sure.

Could anyone guide me on:

What might be causing this issue despite having a valid SSL certificate?
How to systematically troubleshoot and fix this problem?
Are there tools or scripts to automate checks for mixed content or invalid redirects?
I’d greatly appreciate any insights or recommendations from the community. Thank you in advance!

**Comment by Logophoman** [comment]
What web Server are you using? Apache or Nginx? Or are you doing something on windows server? 

Can you provide your Apache2 or Nginx Conf files? I suspect that you might still have the HTTP Route allowed (80) and might not automatically redirect everything to HTTPS...

**Comment by mecodeatlas** [comment]
Hi @LeeKoehn,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the **SSL Support**. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by mecodeatlas** [comment]
And @LeeKoehn! You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!
